aws/resource_aws_instance_test.go

```diff
@@ -236,52 +236,51 @@ func TestAccAWSInstance_inEc2Classic(t *testing.T) {
 
 func TestAccAWSInstance_basic(t *testing.T) {
```

Comment:
Modify `TestAccAWSInstance_basic` to test the minimal configuration.
Move the previous test functionality into new test `TestAccAWSInstance_atLeastOneOtherEbsVolume`.

Reply:
I like this 👍